packages/core/js-client/src/util/bytes.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+type Size = 'u32' | 'u64';
+
+export function numberToBytes(n: number, s: Size, e: boolean) {
```

Comment:
I suggest not to save characters here
using `n` for `number` is probably ok. Maybe `s` for `size` is fine, but why not just use `size` but using `e` - I don't know what `e` is at all. Why not name it `littleEndian` exactly like in standard ts definitions, because that's what it actually is. It also can be an optional value just like in `setUint32` method

Also I just noticed you don't ever use it outside, Do you have a plan to ever use it outside or most probably only `numberToLittleEndianBytes` will be used? I prefer not export until there is a need to export cause it clutters Intellisense with the stuff that you don't need/want to use

Also I would suggest moving `sizeMap` outside cause it's neverchanging constant and there is no need to recreate it on each function envocation

Reply:
thanks, fixed! i created numberToLittleEndianBytes because i don't like to call `numberToBytes` directly and pass a lot of parameters. I consider that function as internal. I prefer making additional functions with more verbose names and less argument list which are public functions.

Reply:
nice job